### DISCUSSION TITLE
Is it possible to make mt-32 music work correctly in ScummVM?

### DISCUSSION THREAD

**Discussion by Starbrush** [OnionUI/Onion#1437] [Q&A]
I discovered the ScummVM system because of the miyoo mini, and I really loved it, I discovered classic games and it's really fun to play on the miyoo, but when I tried to play the mt-32 songs from my games, it was very slow, and I saw that in ScummVM standalone it wasn't includes "emulate mt-32" what I want to know is if the miyoo mini is not powerful enough to run mt-32, or can this be fixed in future versions?

**Comment by Nighted** [comment]
Did you check youtube? There are a few Miyoo/ScummVM videos there. Hope you find your answer there.

I haven't used ScummVM so I have no experience with this.

**Reply by Starbrush** [reply]
unfortunately no, only more experienced people or devs  know to respond about these audio issues, the mt-32 format is wonderful, the songs still sound modern, I was upset that it doesn't run properly,I hope someone can help me with this question!

**Comment by Starbrush** [comment]
I ended up finding out, unfortunately it doesn't run well because the miyoo really isn't powerful enough 😓

**Reply by schmurtzm** [reply]
Did you try the "[Target FPS for stutter reduction](https://onionui.github.io/docs/emulators/scummvm#speed-troubleshooting)" option in retroarch ?
Did you try to play with [overclock](https://onionui.github.io/docs/faq#how-to-overclock-my-miyoo-mini) ?

**Reply by Starbrush** [reply]
I just tried both, I set the overclock to 1800 and the target fps to 15, but the audio is still slow for some reason
